agimus_demos_common/launch/franka_common_lfc.launch.py

```diff
@@ -0,0 +1,125 @@
+from launch import LaunchContext, LaunchDescription
+from launch.actions import (
+    DeclareLaunchArgument,
+    IncludeLaunchDescription,
+    OpaqueFunction,
+    RegisterEventHandler,
+)
+from launch.event_handlers import OnProcessExit
+from launch.launch_description_entity import LaunchDescriptionEntity
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.substitutions import (
+    LaunchConfiguration,
+    PathJoinSubstitution,
+)
+from launch_ros.substitutions import FindPackageShare
+
+from controller_manager.launch_utils import (
+    generate_load_controller_launch_description,  # noqa: I001
+)
+
+from agimus_demos_common.launch_utils import (
+    generate_default_franka_args,
+)
+
+
+def launch_setup(
+    context: LaunchContext, *args, **kwargs
+) -> list[LaunchDescriptionEntity]:
+    franka_robot_launch = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
+            [
+                PathJoinSubstitution(
+                    [
+                        FindPackageShare("agimus_demos_common"),
+                        "launch",
+                        "franka_common.launch.py",
+                    ]
+                )
+            ]
+        ),
+        launch_arguments={
+            "arm_id": LaunchConfiguration("arm_id"),
+            "robot_ip": LaunchConfiguration("robot_ip"),
+            "franka_controllers_params": LaunchConfiguration(
+                "franka_controllers_params"
+            ),
+            "use_gazebo": LaunchConfiguration("use_gazebo"),
+            "use_rviz": LaunchConfiguration("use_rviz"),
+            "rviz_config_path": LaunchConfiguration("rviz_config_path"),
+            "gz_verbose": LaunchConfiguration("gz_verbose"),
+            "gz_headless": LaunchConfiguration("gz_headless"),
+        }.items(),
+    )
+
+    linear_feedback_controller_params = LaunchConfiguration(
+        "linear_feedback_controller_params"
+    )
+
+    load_linear_feedback_controller = generate_load_controller_launch_description(
+        controller_name="linear_feedback_controller",
+        controller_params_file=linear_feedback_controller_params,
+        extra_spawner_args=["--controller-manager-timeout", "1000"],
+    )
+
+    load_joint_state_estimator = generate_load_controller_launch_description(
+        controller_name="joint_state_estimator",
+        controller_params_file=linear_feedback_controller_params,
+        extra_spawner_args=["--controller-manager-timeout", "1000"],
+    )
+
+    return [
+        franka_robot_launch,
+        load_linear_feedback_controller,
+        RegisterEventHandler(
```

Comment:
if you have issues loading the robot model you might want until the joint_state_broadcaster is spawned but I think that in the cpp code of LFC we wait until the parameter becomes available so it might be reasonable doing it like you do.